client/src/components/glossary_components.tsx

```diff
@@ -11,27 +11,52 @@ import { is_a11y_mode } from "src/core/injected_build_constants";
 import { IconQuestion } from "src/icons/icons";
 import { glossary_href } from "src/link_utils";
 
-const GlossaryTooltipWrapper = ({ id, children, no_bottom_border }) =>
+interface GlossaryTooltipWrapperProps {
+  id: string;
+  children?: React.ReactNode;
+  no_bottom_border?: boolean;
+}
+
+interface GlossaryIconProps {
+  id: string;
+  alternate_text?: string;
+  arrow_selector?: any;
+  inner_selector?: any; // i don't know how to do the types for the selectors
+  icon_color?: string;
+  icon_alt_color?: string;
+}
+
+interface GlossaryItemProps {
+  id: string;
+  alternate_text: string;
+  item_class: string;
+}
+
+const GlossaryTooltipWrapper: React.FC<GlossaryTooltipWrapperProps> = ({
+  id,
+  children,
+  no_bottom_border,
+}) =>
   is_a11y_mode ? (
     <a
-      href={is_a11y_mode ? glossary_href(id) : null}
+      href={glossary_href(id)}
       title={is_a11y_mode ? trivial_text_maker("glossary_link_title") : null}
     >
       {children}
     </a>
   ) : (
     <span
       className="nowrap glossary-tippy-link"
-      style={no_bottom_border && { borderBottom: "none" }}
-      tabIndex="0"
+      style={{ borderBottom: no_bottom_border ? "none" : "initial" }}
```

Comment:
Change in behaviour here, is that intentional? Setting the inline style to `borderBottom: "initial"` isn't the same as not setting a `borderBottom` style. 

Also, I didn't know this off the top of my head, but it looks like IE doesn't support the `initial` value https://developer.mozilla.org/en-US/docs/Web/CSS/initial#browser_compatibility

Reply:
Supplement explanation: "initial" will be different from not setting it because `glossary-tippy-link` is setting `border-bottom`

Reply:
Should I replace `"initial"` with `""`?

Reply:
I believe that'd work, but only because it's an invalid `border-bottom` value and would be ignored in the browser. 

Something to look out for: inline styles will be picked over any CSS-applied styles, unless those CSS styles use `!important`. That's what's happening here, there's some CSS applying a `border-bottom` here but the inline `border-bottom: none` will overrule that when set. Otherwise, we want whatever the CSS says, in which case we can't set any inline `border-bottom` value.

Small side note, `!important` is dangerous and should only be used if absolutely necessary (e.g. to overrule styles on some library component that we can't directly edit). It's dangerous _because_ it throws out the normal rules (like "inline styles beat style sheet styles").

Given that, one of these would be better:

1) `style={{ ...(no_bottom_border && { borderBottom: "none" }) }}`

Kind of ugly, but it's a useful trick with object spread `...` to inline a conditional key in an object. If `no_bottom_border` is false, it short circuits to `{ ...(false) }`, and object spread of anything falsey will add nothing to the resulting object.

2) `style={no_bottom_border ? { borderBottom: "none" } : undefined}`
Probably better here, clearer that we're not setting any inline styles when `no_bottom_border` is false. I showed 1) first just because it's a handy trick in other situations (e.g. if there were multiple styles and only some needed to be set conditionally).